docs/uppy-core.mdx

```diff
@@ -794,7 +794,7 @@ const state = {
 	capabilities: {
 		resumableUploads: false,
 	},
-	totalProgress: 0,
+	progress: null,
```

Comment:
Not a fan of this change. Let's keep it `totalProgress` and reduce the tech debt, confusion, and PR diff.